### PR TITLE
LL-1650 Send flow rework 

### PR DESCRIPTION
**Pull request by juan-cortes**
Big rework of the send flow in order to make it friendlier, it involves changes in the whole flow steps, wordings, styles, it removes the double modal for high fees, etc. It also takes advantage of the introduced debouncing of the bridge status so any loading steps will be shown on the continue button.

image
Shows recipient-related fields, in this case, the tag for XRP

image

image
Loading state for the continue button

image
image
New high fees warning, replacing the modal

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -122,6 +122,7 @@
     "styled-components": "^4.3.2",
     "styled-system": "^2.2.9",
     "uncontrollable": "^6.0.0",
+    "use-debounce": "^3.1.0",
```
Comment (gre) [inline package.json]: not used

**Comment by gre** [comment]
Check CI

**Comment by juan-cortes** [comment]
> I love it :)

It is quite a lot nicer to go through :D

**Comment by gre** [comment]
@Arnaud97234 

> To be consistent with mobile we could hide that info from Summary

which info?

> Fees amount should be displayed using gwei to ETH conversion instead of Gwei and using the corresponding unit (ETH instead of ENG for instance)

I think in fact we need to diverge away from this idea.
- We should use the same unit for all the summary part (the user account prefered unit) so user can easily "do the math".
- the device step MUST be in the currency unit because that's what displayed on the device.

**Comment by gre** [comment]
sorry, i misunderstood that.

**Comment by juan-cortes** [comment]
@Arnaud97234 current version should show the correct units for the erc20 tokens

image
image

**Pull request merged**